src/centralstore/incremental/NodeManager.cpp

```diff
@@ -118,6 +118,7 @@ RelationBlock *NodeManager::addRelation(NodeBlock source, NodeBlock destination)
 
 NodeBlock *NodeManager::addNode(std::string nodeId) {
     unsigned int assignedNodeIndex;
+    std::cout << "################# node index = " << this->nextNodeIndex << std::endl;
```

Comment:
This is kind of debug log right? Rather than use of `std::cout` can we use the logger?

Reply:
Yes change it to a debug log in this commit https://github.com/miyurud/jasminegraph/pull/132/commits/d8d887f5453408266df16065d675be6691184b72

Reply:
Ok